### PR TITLE
Hedgewars version bump

### PR DESCRIPTION
**Pull request by sepi**

### CODE REVIEW

File: pkgs/games/hedgewars/default.nix [hunk 1]
```diff
@@ -1,24 +1,23 @@
+
 { SDL_image, SDL_ttf, SDL_net, fpc, qt4 , ghc, ffmpeg, freeglut, network, vector
 , stdenv, makeWrapper, fetchurl, cmake, pkgconfig, lua5_1, SDL, SDL_mixer
-, utf8String, bytestringShow, hslogger, random, dataenc, zlib, libpng, mesa
+, utf8String, bytestringShow, hslogger, random, dataenc, zlib, libpng, mesa, haskellPackages, physfs
```
Comment (jagajaga) [inline pkgs/games/hedgewars/default.nix]: Aren't `haskellPackages` deprecated, @peti?

**Comment by jagajaga** [comment]
Have you confused the title?

File: pkgs/games/hedgewars/default.nix (hunk 1 above)
Comment (peti) [inline pkgs/games/hedgewars/default.nix]: Yes, ` haskellPackages` is dead'ish and will be removed in 2-3 weeks. `haskellngPackages` is the way to go.

Comment (jagajaga) [inline pkgs/games/hedgewars/default.nix]: So, @sepi, please use `haskellngPackages`.

**Pull request closed without merging**